tool/cli.js

```diff
@@ -891,6 +893,61 @@ if (Mozilla && !Mozilla.dntEnabled()) {
     })
   )
 
+  .command(
+    "frontmatter-inventory",
+    "Create frontmatter content inventory as JSON"
+  )
+  .argument("<output>", "Output to file or stdout", {
+    default: "stdout",
+  })
+  .action(
+    tryOrExit(async ({ args, options }) => {
+      const { verbose } = options;
+      const { output } = args;
+
+      if (verbose) {
+        console.log(chalk.green("Checking for precense of CONTENT_ROOT"));
```

Comment:
The verbose output still writes to stdout. If you run `yarn tool frontmatter-inventory file --verbose` then it produces malformed JSON

```suggestion
        console.error(chalk.green("Checking for precense of CONTENT_ROOT"));
```